File: src/main/java/org/jukeboxmc/network/handler/ModalFormResponseHandler.java
package org.jukeboxmc.network.handler;

import com.nukkitx.protocol.bedrock.packet.ModalFormResponsePacket;
import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.Server;
import org.jukeboxmc.player.Player;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ModalFormResponseHandler implements PacketHandler<ModalFormResponsePacket> {

    @Override
    public void handle(@NotNull ModalFormResponsePacket packet, Server server, @NotNull Player player ) {
        player.parseGUIResponse( packet.getFormId(), packet.getFormData() );
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/network/handler/ModalFormResponseHandler.kt
package org.jukeboxmc.server.network.handler

import org.cloudburstmc.protocol.bedrock.packet.ModalFormResponsePacket
import org.jukeboxmc.server.JukeboxServer
import org.jukeboxmc.server.player.JukeboxPlayer

/**
 * @author Kaooot
 * @version 1.0
 */
class ModalFormResponseHandler : PacketHandler<ModalFormResponsePacket> {

    override fun handle(packet: ModalFormResponsePacket, server: JukeboxServer, player: JukeboxPlayer) {
        player.parseGUIResponse(packet.formId, packet.formData)
    }
}